apps/backend/package.json

```diff
@@ -88,7 +88,9 @@
     "lint-staged": "13.1.0",
     "prettier": "2.8.3",
     "supertest": "6.3.3",
+    "terser-webpack-plugin": "^5.3.9",
```

Comment:
one question: In previous projects, I have been asked to remove the caret prefix on the package version in order to lock it to the exact minor/patch version. Is this still a preferred practice? It seems like leaving the caret prefix and allowing for minor/patch updates would not be breaking?

Reply:
Updated to lock it down.

Yeah I try to lock them down as well. *Most* of the time minor / patch updates don't break things, buuuut from experience not always 😅